lang/es.js

```diff
@@ -91,7 +91,7 @@ var language = {
       'update-req': 'Necesitas <a target="_blank" href="https://support.microbit.org/support/solutions/articles/19000019131-how-to-upgrade-the-firmware-on-the-micro-bit">actualizar el firmware del micro:bit </a> para poder usar esta función!',
       'clear-connect': 'Hay otro proceso conectado a este dispositivo.<br>Cierre cualquier otra pestaña que pueda estar usando WebUSB (por ejemplo, MakeCode or Editor de Python), o desconecte y vuelva a conectar el micro:bit antes de volver a intentarlo.',
       'reconnect-microbit': 'Por favor reconecta el micro:bit e intentalo de nuevo.',
-      'partial-flashing-disable': 'Si el error persiste, intenta deshabilitar el flasheo parcial en las opciones beta.',
+      'partial-flashing-disable': 'Si el error persiste, intenta deshabilitar el flasheo rapido en las opciones beta.',
```

Comment:
Perfect Spanish!

We also need to update the `options-dropdown` > `partial-flashing` entry in line 188.